src/XrdHeaders.cmake

```diff
@@ -125,6 +125,18 @@ set( XROOTD_PRIVATE_HEADERS
   XrdOfs/XrdOfsHandle.hh
   XrdOfs/XrdOfsTrace.hh
   XrdOfs/XrdOfsTPCInfo.hh
+  XrdSsi/XrdSsiAtomics.hh
```

Comment:
The only reason these are in the private section is because the Castor people wanted some additional enhancements that may or may not break ABI. So, this is temporary until we get that straightened out. After that the headers will go into the public section.